README.md

```
@@ -1 +1,157 @@
+### Run Locally with Docker
+
+```bash
+# Using Make (recommended)
+make dev
+
+# Or manually
+docker build -t onitama-web .
+docker run -d --name onitama-web-container -p 3000:3000 -v $(pwd)/data:/app/data onitama-web
+```
+
+Then open your browser to http://localhost:3000
```

markdownlint-cli2 (0.20.0)

Bare URL used

(MD034, no-bare-urls)